source/ways-of-working/new-component/LaunchDarkly.html.md.erb

```diff
@@ -0,0 +1,21 @@
+---
+title: Feature flags
+last_reviewed_on: 2021-10-29
+review_in: 6 months
+weight: 99
+---
+
+# <%= current_page.data.title %>
+
+Feature flags are a software development and delivery technique that allows software teams to enable and disable parts of a codebase at the flip of a switch.
+Feature flags enable software teams to adopt DevOps practices, in so much as they help increase deployment speeds, system stability, and cross-team collaboration.
+Feature flags also allow you to decouple code deployments from feature releases. You can make code changes in production while hiding those changes from users.
+
+### Accessing LaunchDarkly.
```

Comment:
Needs to include how to setup a new project:
https://github.com/hmcts/launchdarkly-terraform

and should have a section on Environments, how we only have the testing and production environments. generally flags should be globally set the same as in production and user targeting should be used apart from that.